docker-entrypoint.sh

```diff
@@ -32,21 +32,35 @@ elif [[ "${CMD}" = "${JOB_MANAGER}" || "${CMD}" = "${JOB_CLUSTER}" || "${CMD}" =
     shift
     sed -i -e "s/jobmanager.rpc.address: localhost/jobmanager.rpc.address: ${JOB_MANAGER_RPC_ADDRESS}/g" "$FLINK_HOME/conf/flink-conf.yaml"
     echo "metrics.internal.query-service.port: ${CONTAINER_METRIC_PORT}" >> "$FLINK_HOME/conf/flink-conf.yaml"
```

Comment:
Bash tip: use this instead:

```bash
cat <<EOF >> "$FLINK_HOME/conf/flink-conf.yaml"
metrics.internal.query-service.port: ${CONTAINER_METRIC_PORT}
rest.port: 8081
blob.server.port: 6124
query.server.port: 6125
EOF
```

and similarly for the subsequent sections. I wouldn't change it now, though...